CHANGELOG.md

```diff
@@ -1,5 +1,3 @@
-# Changelog
-
```

Comment:
This diff in changelog doesn't seem correct. @maheshrajamani can you please help with this?

Reply:
@maheshrajamani @tatu-at-datastax Do you have any idea how to fix this?

Reply:
I think these need to be changed by manually editing; I am not sure generation can be fixed.
But I am not sure why everything went into "Unreleased" -- maybe you can just change "Unreleased" header with copy-modified one that was done for `2.0.9` (but changing references)?

So like

```
## [v1.0.0-ALPHA-10](https://github.com/stargate/jsonapi/tree/v1.0.0-ALPHA-10) (2023-07-??)

[Full Changelog](https://github.com/stargate/jsonapi/compare/v1.0.0-ALPHA-9...v1.0.0-ALPHA-10)

(existing entries)
```